<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsView</class>
 <widget class="QMainWindow" name="SettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="rightMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>12</number>
      </property>
      <property name="spacing">
       <number>8</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="cameraPortCb"/>
      </item>
      <item row="3" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lowRedValueText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>LOW RED [170, 0, 0] [180, 255, 255]
MEDIAN BLUR 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="highRedValueText">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>HIGH RED [0, 0, 0] [30, 255, 255]
MEDIAN BLUR 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeDetectionParamsBtn">
          <property name="text">
           <string>Change Value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cameraResText">
        <property name="text">
         <string>Camera Resolution</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="separator">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="noiseReductionTypeText">
        <property name="text">
         <string>Noise Reduction Type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="noiseReductionTypeCb">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="detectionParamsText">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>Detection Params</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cameraResCb"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="cameraPortText">
        <property name="text">
         <string>Camera Port</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controlLayout">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string> Save</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/diskette.png</normaloff>Resources/diskette.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string> Close</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Resources/delete-button.png</normaloff>Resources/delete-button.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
